### PR TITLE
Header cleanup

### LINKED ISSUE #52: Header syntax
Must as with server-timing, NEL is NOT defined as a comma-delimited field. If we want multiple instances of the header to be combined, e.g.,

    NEL: report-URI="/"
    NEL: max-age=3600

then it needs to be comma-delimited. Otherwise, if we really want to use semicolons, we'll need to define error handling for multiple fields.

Also, it would be good to strongly caution people to quote report-uri when producing the header, so that there aren't issues with delimiters in URIs.

Do we really want the registry of directives to be under IETF Review rules? That seems odd, considering the W3C is producing this spec; also, the IETF may not want to...

See also:
  http://httpwg.github.io/specs/rfc7231.html#considerations.for.new.header.fields

### PR DESCRIPTION
**Pull request by igrigorik**
Cleanup header definition and processing, closes #52.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,14 +124,18 @@
...
-          <li>User agents MUST ignore any <a>NEL header field</a> containing directives, or other header field value data, that does not conform to the syntax defined in this specification.</li>
-          <li>If an <a>NEL header field</a> contains directive(s) not recognized by the user agent, the user agent MUST ignore the unrecognized directives, and if the <a>NEL header field</a> otherwise satisfies the above requirements, the user agent MUST process the recognized directives.</li>
+          <li>Directive order is not significant.</li>
+          <li>Directive should be specified once.</li>
         </ul>
 
-        <p>Additional directives extending the semantic functionality of the <a>NEL header field</a> can be defined in other specifications, with a registry (having an IANA policy definition of IETF Review [[!RFC5226]]) defined for them at such time.</p>
+        <p>User agents MUST ignore any unknown or invalid directive(s), or other header field value data, that does not conform to the syntax defined in this specification. A valid <a>NEL header field</a> MUST, at a minimum, contain all of the "REQUIRED" directives defined in this specification.</p>
```
Comment (mnot) [inline index.html]: Before, it was MUST IGNORE a field containing invalid syntax; now it's MUST IGNORE the invalid data - is that an intentional change?

Comment (igrigorik) [inline index.html]: FWIW, the text is the same, just in a different location -- see https://github.com/w3c/network-error-logging/pull/53/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL131.

"User agents MUST ignore any <a>NEL header field</a> containing directives, or other header field value data, that does not conform to the syntax defined in this specification."

**Comment by igrigorik** [comment]
@mnot updated to use double-quotes, see: https://github.com/w3c/network-error-logging/commit/ec67bcf9003d4261d2c8fe5069143fd7ab6a86f3. 

Any objections to merging this? Anything else missing?

**Comment by mnot** [comment]
LGTM. Thanks.

**Pull request merged**